cmd/manager/main.go

```diff
@@ -71,12 +71,13 @@ func main() {
 	// Initialize cluster actuator.
 	clusterActuator := cluster.NewActuator(cluster.ActuatorParams{
 		Client:         cs.ClusterV1alpha1(),
-		LoggingContext: "[cluster actuator]",
+		LoggingContext: "[\033[32mcluster actuator\033[0m]",
```

Comment:
the color makes it super easy to read, but I'm ok if this isn't good for whatever reason

Reply:
I do like the idea of adding color, but I'm not necessarily sure we can rely on the user's shell supporting it.

Reply:
This should probably be configurable